platform/bitcoin/api_test.go

```diff
@@ -126,16 +146,41 @@ func TestInferDirection(t *testing.T) {
 		t.Error("direction is not outgoing")
 	}
 
-	inputs2 := []string{
-		"3CgvDkzcJ7yMZe75jNBem6Bj6nkMAWwMEf",
-		"3LyzYcB54pm9EAMmzXpFfb1kzEDAFvqBgT",
-		"3Q6DYour5q5WdMhyXsyPgBeAqPCXchzCsF",
-		"3JZZM1rwst7G5izxbFL7KNvy7ZiZ47SVqG",
+	inputs2 := []blockatlas.TxOutput{
```

Comment:
We can do a table test here, its easier and we don't need to make the logic again for more than one case.

This is a sample:
https://github.com/golang/go/wiki/TableDrivenTests